querybook/webapp/lib/sql-helper/sql-autocompleter.ts

```diff
@@ -240,9 +240,12 @@ export class SqlAutoCompleter {
             return [];
         }
 
+        let omit_schema = this.codeAnalysis.lineage.activeSchema || 'default';
```

Comment:
`const`.  also `|| default` is not necessarily needed, as the API has the default value `default`  for the 3rd parameter.  could be just below
```
suggest(
            metastoreId,
            prefix,
            this.codeAnalysis.lineage.activeSchema)
```